src/sql/workbench/contrib/notebook/test/browser/notebookLinkHandler.test.ts

```diff
@@ -130,4 +130,21 @@ suite('Noteboook Link Handler', function (): void {
 			assert.strictEqual(notebookLinkHandler.getEncodedLinkUrl(), `/Notebooks/Test_Paths/My%2520File.ipynb`, '% in file path failed to be encoded');
 		});
 	});
+	test('getLinkUrl should return relativePath correctly', () => {
```

Comment:
Add tests with : 

- Non-encoded spaces
- Encoded characters in the path (for example a path with a folder named `Test%20Folder` in it)

Reply:
the tests are already there for them: https://github.com/microsoft/azuredatastudio/blob/0e65f0d7eb555d6d9ae8eb5504965b7f0f65afd1/src/sql/workbench/contrib/notebook/test/browser/notebookLinkHandler.test.ts#L104